modules/administration/pages/reports.adoc

```diff
@@ -23,68 +34,94 @@ To get the report in CSV format, run this command at the command prompt on the s
 spacewalk-report <report_name>
 ----
 
+[[spacewalk-report-reporting-db]]
+== ``spacewalk-report`` and the reporting database
+
+[command]``spacewalk-report`` uses by default the new reporting database to extract the data.
+
+This means that the new generated reports will have some differences in the structure and the format of the data.
+
+The differences common to all reports are:
+
+- the report data is not changing in real-time, but it's updated only by the execution of a scheduled task;
+- data duplication has been removed and columns that were previously considered "multival" contains now multiple values separated by ``;``. This also means that the command line options ``--multival-on-rows`` and ``--multival-separator`` are no longer applicable to the new reports, as their behavior is now the default;
+- in all reports the new columns ``mgm_id`` and ``synced_date`` have been introduced to identify respectively the management server in the hub scenario and the last time the information was updated from the application database;
+- all boolean values are now represented by the ``True``/``False`` and not by a ``1``/``0`` values;
+- the column ``org_id`` has been replaced by ``organization``, which contains the organization name and not the numerical identifier.
+- The term "server" has been replaced by "system". So, for example, the column ``server_id`` is now called ``system_id``.
+
+For report specific changes, please refer to the section <<report-list>>
+
+[IMPORTANT]
+====
+If these differences are problematic, the new option ``--legacy-report`` can be used to fall back to the old report which is executed against the application database.
```

Comment:
```suggestion
If this changed behavior causes trouble, the new option ``--legacy-report`` can be used to fall back to the old report, which is executed against the application database.
```
I'm not sure whether I got the meaning right.

Reply:
Yes, that's better :+1: